CMakeLists.txt

```diff
@@ -108,7 +117,9 @@ pybind11_add_module(idaklu
   ${IDAKLU_EXPR_IREE_SOURCE_FILES}
 )
 
-if (NOT DEFINED USE_PYTHON_CASADI)
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+  set(USE_PYTHON_CASADI FALSE)
+else()
```

Comment:
Yes. This behaviour was previously handled using an environment variable `USE_PYTHON_CASADI`, which converted it to a CMake option `-DUSE_PYTHON_CASADI` that was passed to the CMake command in a subprocess inside `setup.py`. We now set it directly here because we no longer have a `setup.py` file.